steps/gitignore.ts

```diff
@@ -1,10 +1,11 @@
 import type { Init } from "../init.ts";
 
-export default function () {
+export default function() {
   return ({ files }: Init) => {
     const paths = [
-      "_site",
-      "_cache",
+      "# Ignore IDEA settings \n .vscode \n .idea \n",
+      "# Ignore the output directory \n _site \n",
+      "# Ignore the cache directory \n _cache \n",
```

Comment:
This creates extra spaces after and before the paths.
Use an array element per line. For example:


```js
  "# Ignore the output directory",
  "_site",
  "# Ignore the cache directory",
  "_cache",
```